webhookURLValidator.go

```diff
@@ -30,13 +30,14 @@ var (
 	errInvalidFailureURL     = errors.New("invalid Failure URL")
 	errInvalidAlternativeURL = errors.New("invalid Alternative URL(s)")
 	errURLIsNotHTTPS         = errors.New("URL scheme is not HTTPS")
-	errInvalidHost           = errors.New("invalid host")
+	errInvalidHost           = errors.New("host is blocked")
 	errIPGivenAsHost         = errors.New("cannot use IP as host")
 	errLoopbackGivenAsHost   = errors.New("cannot use loopback host")
 	errIPinInvalidSubnets    = errors.New("IP is within a blocked subnet")
 	errInvalidSubnet         = errors.New("invalid subnet")
 	errNoSuchHost            = errors.New("host does not exist")
 	errBadURLProtocol        = errors.New("bad URL protocol")
+	errEmptyURL              = errors.New("error cannot be an empty string")
```

Comment:
I think you mean "config.url cannot be an empty string"

Reply:
Oops 😅